[EWS] Never retry no-change build for pull-requests

JonWBedard opened the pull request:
#### 61fdbc23a37f16473180863ad571bfa5b754361f
<pre>
[EWS] Never retry no-change build for pull-requests
<a href="https://bugs.webkit.org/show_bug.cgi?id=236046">https://bugs.webkit.org/show_bug.cgi?id=236046</a>
&lt;rdar://problem/88408671 &gt;

Reviewed by NOBODY (OOPS!).

Pull-requests are not automatically rebased in EWS, so if a build fails on a specific
commit, a retry is unlikely to rectify the situation.

* Tools/CISupport/ews-build/steps.py:
(ReRunWebKitTests.evaluateCommand):
(RunWebKitTestsRedTree.evaluateCommand):
(RunWebKitTestsRepeatFailuresRedTree.evaluateCommand):
(ReRunAPITests.evaluateCommand):
</pre>

aj062 reviewed (changes requested):
r-. This change isn't required for reason explained in https://github.com/WebKit/WebKit/pull/115#discussion_r798123188

The retry done in this step is not about rebasing/ToT/testing-on-top-of-different-revision. It is simply to retry on different bot. Ideal solution would be to also reboot that bot while retrying the build (Tracked in https://bugs.webkit.org/show_bug.cgi?id=235480).

JonWBedard committed:
[EWS] Never retry no-change build for pull-requests
https://bugs.webkit.org/show_bug.cgi?id=236046
<rdar://problem/88408671>

Reviewed by NOBODY (OOPS!).

Pull-requests are not automatically rebased in EWS, so if a build fails on a specific
commit, a retry is unlikely to rectify the situation.

* Tools/CISupport/ews-build/steps.py:
(ReRunWebKitTests.evaluateCommand):
(RunWebKitTestsRedTree.evaluateCommand):
(RunWebKitTestsRepeatFailuresRedTree.evaluateCommand):
(ReRunAPITests.evaluateCommand):
